jdi-light/src/main/java/com/epam/jdi/light/settings/PageSettings.java

```diff
@@ -3,9 +3,49 @@
 import com.epam.jdi.light.common.PageChecks;
```

Comment:
revert changes in the class, it was need to research

Reply:
Done